.github/workflows/coverage.yml

```diff
@@ -36,6 +36,14 @@ jobs:
       - name: Install package
         run: python -m pip install '.[test]'
 
+      - name: Clone acc-models
+        run: |
+          git clone -b 2023 https://gitlab.cern.ch/acc-models/acc-models-lhc.git acc-models-lhc-2023
+          git clone -b 2022 https://gitlab.cern.ch/acc-models/acc-models-lhc.git acc-models-lhc-2022
+          git clone -b 2018 https://gitlab.cern.ch/acc-models/acc-models-lhc.git acc-models-lhc-2018
+          git clone -b 2021 https://gitlab.cern.ch/acc-models/acc-models-psb.git acc-models-psb-2021
+          git clone -b 2021 https://gitlab.cern.ch/acc-models/acc-models-ps.git acc-models-ps-2021
+
```

Comment:
I would probably do these as fixtures, so that you can easily run the tests also offline, without having to clone manually.

Reply:
I'd also like some input from @fsoubelet on this, maybe he has an opinion.
My suggestion (untested):

add `gitpython` as test requirement.
in `test/conftest.py`:

```python
import git
import pytest

GITLAB_REPO_ACC_MODELS_LHC = "https://gitlab.cern.ch/acc-models/acc-models-lhc.git"

@pytest.fixture(scope="session")
def acc_models_lhc_2023(tmp_path_factory):
    return acc_models_lhc(tmp_path_factory, 2023)

def acc_models_lhc(tmp_path_factory, year: int):
    tmp_path = tmp_path_factory.mktemp(f"acc-models-lhc-{year}")
    git.Repo.clone_from(GITLAB_REPO_ACC_MODELS_LHC , tmp_path, branch=str(year)) 
```

Reply:
done!